lib/src/controls/better_player_controls_state.dart

```diff
@@ -456,6 +456,7 @@ abstract class BetterPlayerControlsState<T extends StatefulWidget>
     showCupertinoModalPopup<void>(
       barrierColor: Colors.transparent,
       context: context,
+      useRootNavigator: false,
```

Comment:
This should be definitely parametrized in BetterPlayerConfiguration. This solves your issue, but it may break other people apps.

Please add configuration parameter for `useRootNavigator`.